.circleci/config.yml

```diff
@@ -8,3 +8,26 @@ jobs:
       - run:         
           name: Run Tests
           command:  docker-compose run web pytest --cov-config=.coveragerc --cov=ops_site --cov-report term-missing
+      - run:
+          name: Install cf cli
+          command: |
+            mkdir -p $HOME/bin
+            export PATH=$HOME/bin:$PATH
+            curl -L "https://cli.run.pivotal.io/stable?release=linux64-binary" | tar xzv -C $HOME/bin
+            cf install-plugin autopilot -f -r CF-Community
+      - run:
+          name: Deploy to cloud.gov
+          filters:
+            tags:
+              only: /^v*/
```

Comment:
[issue] I don't think this belongs here.  `filters` are for workflow/jobs, so it goes to a level up.  And I think this is not correct either to only look for tags that begins with `v`.
[suggestion] This should be removed.